data/json/monsters/reptile_amphibian.json

```diff
@@ -1,4 +1,52 @@
 [
+  {
+    "id": "mon_frog",
+    "type": "MONSTER",
+    "name": { "str": "bullfrog" },
+    "description": "The American bullfrog, in its natural habitat.  It feeds on insects, mice, lizards and any other living thing it can stuff down its gullet.",
+    "default_faction": "frog",
+    "bodytype": "frog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "AMPHIBIAN" ],
+    "volume": "750 ml",
+    "weight": "1 kg",
+    "hp": 3,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "b",
+    "color": "green",
+    "melee_cut": 0,
+    "dodge": 6,
+    "reproduction": { "baby_egg": "egg_frog", "baby_count": 8, "baby_timer": 26 },
+    "harvest": "mammal_tiny",
+    "special_attacks": [ { "type": "leap", "cooldown": 10, "max_range": 5 } ],
+    "fear_triggers": [ "PLAYER_CLOSE" ],
+    "flags": [ "SEES", "SMELLS", "HEARS", "SWIMS", "WATER_CAMOUFLAGE" ]
+  },
+  {
+    "id": "mon_tadpole",
+    "type": "MONSTER",
+    "name": { "str_sp": "tadpole" },
+    "description": "An amphibian in its larval stage.  It will eventually grow legs and start breathing air.",
+    "default_faction": "frog",
```

Comment:
It should be instructive to see how well WATER_CAMOUFLAGE protects these against various things that will attack them, with this faction. If it isn't sufficient, this will need to be changed to something based on fish with frog (probably) as FRIENDLY to it and vice-versa.